jihyeon/models/likesDao.js

```diff
@@ -0,0 +1,68 @@
+const { DataSource } = require("typeorm");
+
+const dataSource = new DataSource({
+  type: process.env.DB_CONNECTION,
+  host: process.env.DB_HOST,
+  port: process.env.DB_PORT,
+  username: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+});
+
+dataSource
+  .initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!");
+  })
+  .catch((err) => {
+    console.log("Error during Data Source initialization", err);
+  });
+
+const likes = async (userId, postId) => {
+  try {
+    const isExists = await dataSource.query(
+      `
+                SELECT EXISTS (
+                    SELECT * FROM likes 
+                    WHERE (
+                      user_id = ? AND post_id = ?
+                    )
+                )
+        `,
+      [userId, postId]
+    );
+
+    if (Object.values(isExists[0]) == 1) {
+      await dataSource.query(
+        `
+                DELETE FROM likes
+                WHERE user_id = ? AND post_id = ?
+          `,
+        [userId, postId]
+      );
+      return false;
+    } else {
+      await dataSource.query(
+        `
+                INSERT INTO likes (
+                    user_id,
+                    post_id
+                ) VALUES (
+                    ?,
+                    ?
+                )
+          `,
+        [userId, postId]
+      );
+      return true;
+    }
```

Comment:
```suggestion
    } 
    await dataSource.query(
      `
              INSERT INTO likes (
                  user_id,
                  post_id
              ) VALUES (
                  ?,
                  ?
              )
          `,
        [userId, postId]
      );
    return true;
```
위에서 언급드린것과 마찬가지로 else는 가급적 쓰지 않도록 해주세요~ 어차피 if문 조건에 해당되거나 아니거나 이기 때문입니다.